examples/app/uploadservice-s3/src/main/java/com/example/uploadservice/s3/S3UploadRequest.kt

```diff
@@ -0,0 +1,38 @@
+package com.example.uploadservice.s3
```

Comment:
suggest using same path as ftp module
uploadservice-ftp/src/main/java/net/gotev/uploadservice/ftp.
instead of com/example